tests/services/test_planning_svc.py

```diff
@@ -215,6 +222,18 @@ def test_get_cleanup_links(self, loop, setup_planning_test, planning_svc):
         assert link_list[0].command == ability.cleanup[0]
 
     def test_generate_and_trim_links(self, loop, setup_planning_test, planning_svc):
-        ability, agent, operation = setup_planning_test
+        ability, agent, operation, _ = setup_planning_test
         generated_links = loop.run_until_complete(planning_svc.generate_and_trim_links(agent, operation, [ability]))
         assert 1 == len(generated_links)
+
+    def test_link_fact_coverage(self, loop, setup_planning_test, planning_svc):
```

Comment:
Did we want to have this unit test here or in its own `test_base_planning_svc.py` module?